src/Workspaces/SharedUtilitiesAndExtensions/Compiler/Core/Services/SyntaxFacts/AbstractSyntaxFacts.cs

```diff
@@ -33,27 +33,6 @@ protected AbstractSyntaxFacts()
         {
         }
 
-        private bool IsWhitespaceTrivia(SyntaxTrivia trivia)
-            => SyntaxKinds.WhitespaceTrivia == trivia.RawKind;
-
-        private bool IsEndOfLineTrivia(SyntaxTrivia trivia)
-            => SyntaxKinds.EndOfLineTrivia == trivia.RawKind;
-
-        public bool IsSingleLineCommentTrivia(SyntaxTrivia trivia)
-            => SyntaxKinds.SingleLineCommentTrivia == trivia.RawKind;
-
-        public bool IsMultiLineCommentTrivia(SyntaxTrivia trivia)
-            => SyntaxKinds.MultiLineCommentTrivia == trivia.RawKind;
-
-        public bool IsSingleLineDocCommentTrivia(SyntaxTrivia trivia)
-            => SyntaxKinds.SingleLineDocCommentTrivia == trivia.RawKind;
-
-        public bool IsMultiLineDocCommentTrivia(SyntaxTrivia trivia)
-            => SyntaxKinds.MultiLineDocCommentTrivia == trivia.RawKind;
-
-        public bool IsShebangDirectiveTrivia(SyntaxTrivia trivia)
-            => SyntaxKinds.ShebangDirectiveTrivia == trivia.RawKind;
```

Comment:
moved to extensions.  tehre is no instance data needed to answer these questions.   fits new pattern.